src/redux/pattern/pattern.sagas.js

```diff
@@ -0,0 +1,19 @@
+import { takeEvery, call, put } from 'redux-saga/effects';
+import { setPatterns } from './pattern.actions';
+
+import { getAllPatterns } from './pattern.operations';
+
+import { GET_PATTERNS } from './pattern.types';
+
+export function* handlePatternsLoad({ payload }) {
+  try {
+    const patterns = yield call(getAllPatterns, 0, 1000);
+    yield put(setPatterns(patterns.items));
+  } catch (error) {
+    console.log(error);
```

Comment:
You can use `handlePatternErrors` saga to set errors and redirect user to error page.

Reply:
done